src/elementPlugins/SummaryViewWrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { ErrorBoundary, pluginProvider } from '.'
+import { Header } from 'semantic-ui-react'
+import { SummaryViewWrapperProps, PluginComponents } from './types'
+
+const SummaryViewWrapper: React.FC<SummaryViewWrapperProps> = (props) => {
+  const { element, value } = props
+  const {
+    parameters,
+    category,
+    code,
+    elementTypePluginCode: pluginCode,
+    isEditable,
+    isRequired,
+    isVisible,
+  } = element
+
+  if (!pluginCode || !isVisible || category === 'INFORMATION') return null
```

Comment:
Maybe a good idea to add a comment here about why we are not rendering the INFORMATION type.
Also would be good to use this type as part of a map e.g.:
```
const questionType: { [key: string]: TemplateElementCategory } = {
  INFORMATION: 'INFORMATION',
  QUESTION: 'QUESTION' 
}
```

Or if that's simpler (actually I think this is simpler really) just import the `TemplateElementCategory` and use as `TemplateElementCategory.INFORMATION`. That's better that spreading the string. Harder to do maintanence or check for any type problems in the future.

Reply:
Okay, good suggestion, ta. Done.